src/app/libp2p_helper/src/libp2p_helper/main.go

```diff
@@ -1092,6 +1098,9 @@ func (ap *beginAdvertisingMsg) run(app *app) (interface{}, error) {
 		//       caused by this prometheus issues.
 		// go app.checkBandwidth(id)
 		// go app.checkLatency(id)
+		// go app.checkPeerCount(id)
+		// go app.checkMessageExchanged(id)
+		// go app.checkMessageStats(id)
```

Comment:
are we checking these in commented out intentionally?

Reply:
Yes, we plan to enable these in the near future. This tracks all the functions we need to call here.